UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Helpers/NMHelper.cs

```diff
@@ -41,6 +41,40 @@ public static List<ShowFilesResponseModel> ListFilesResponse(BatchSearchResponse
             return listshowFilesResponseModels;
         }
 
+        public static List<ShowFilesResponseModel> GetShowFilesResponseModel(List<BatchDetails> batchDetails)
+        {
```

Comment:
GetShowFilesResponseModel() method has the repetitive code. Please refactor it.

Reply:
This will be done as a part of PBI 45758